plugins/jobtype/test/azkaban/jobtype/TestHadoopTuningConfigurationInjector.java

```diff
@@ -0,0 +1,67 @@
+package azkaban.jobtype;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.junit.Assert;
+import org.junit.Test;
+
+import azkaban.flow.CommonJobProperties;
+import azkaban.utils.Props;
+
+import com.google.common.io.Files;
+
+
+public class TestHadoopTuningConfigurationInjector {
+
+  @Test
+  public void testTuningErrorHandler() throws IOException {
+
+    File root = Files.createTempDir();
+
+    Props props = new Props();
+    props.put(HadoopConfigurationInjector.INJECT_PREFIX + "mapreduce.task.io.sort.mb", "100");
```

Comment:
should use the Hadoop constants for these keys

Reply:
Done.